apps/aecontract/test/aecontract_SUITE.erl

```diff
@@ -434,6 +441,40 @@ create_contract_init_error(_Cfg) ->
                  aec_accounts:balance(aect_test_utils:get_account(PubKey, S2))),
     ok.
 
+create_contract_init_error_no_create_account(Cfg) ->
+    S  = aect_test_utils:new_state(),
+    S0 = aect_test_utils:setup_miner_account(?MINER_PUBKEY, S),
+    {PubKey, S1} = aect_test_utils:setup_new_account(S0),
+    PrivKey      = aect_test_utils:priv_key(PubKey, S1),
+
+    {ok, Code} = compile_contract(init_error),
+    Overrides = #{ code => Code
+                 , call_data => make_calldata_from_code(Code, <<"init">>, {<<123:256>>, 0})
+                 , gas => 10000
+                 , gas_price => aec_test_utils:min_gas_price()
+                 },
+
+    Tx = create_tx(PubKey, Overrides, S1),
+
+    ForkHeights   = ?config(fork_heights, Cfg),
+    MinervaHeight = maps:get(fortuna, ForkHeights) - 1,
+    FortunaHeight = maps:get(fortuna, ForkHeights) + 1,
+
+    {ok, S2Minerva} = sign_and_apply_transaction(Tx, PrivKey, S1, MinervaHeight),
+    {ok, S2Fortuna} = sign_and_apply_transaction(Tx, PrivKey, S1, FortunaHeight),
+
+    %% Check that the contract is not created
+    ContractKey = aect_contracts:compute_contract_pubkey(PubKey, aetx:nonce(Tx)),
+    ?assertMatch({none, _}, lookup_contract_by_id(ContractKey, S2Minerva)),
+    ?assertMatch({none, _}, lookup_contract_by_id(ContractKey, S2Fortuna)),
+
+    %% In Minerva, the failing init should     create the contract account.
+    %% In Fortuna, the failing init should NOT create the contract account.
```

Comment:
I see this is about the contract account - not the contract - so it is not caught by test case `create_contract_init_error`. Ok.